diff --git a/node_modules/body-parser/lib/types/urlencoded.js b/node_modules/body-parser/lib/types/urlencoded.js
index b2ca8f1..770fe51 100644
--- a/node_modules/body-parser/lib/types/urlencoded.js
+++ b/node_modules/body-parser/lib/types/urlencoded.js
@@ -103,14 +103,14 @@ function urlencoded (options) {
 
     // assert charset
     var charset = getCharset(req) || 'utf-8'
-    if (charset !== 'utf-8') {
-      debug('invalid charset')
-      next(createError(415, 'unsupported charset "' + charset.toUpperCase() + '"', {
-        charset: charset,
-        type: 'charset.unsupported'
-      }))
-      return
-    }
+    // if (charset !== 'utf-8') {
+    //   debug('invalid charset')
+    //   next(createError(415, 'unsupported charset "' + charset.toUpperCase() + '"', {
+    //     charset: charset,
+    //     type: 'charset.unsupported'
+    //   }))
+    //   return
+    // }
 
     // read
     read(req, res, next, parse, debug, {
